src/Simulation/CsharpGeneration.Tests/SimulationCodeTests.fs

```diff
@@ -3204,3 +3203,112 @@ namespace Microsoft.Quantum.Tests.LineNumbers
         |> 
         testOneFile (Path.Combine("Circuits","LineNumbers.qs"))
 
+        
+    [<Fact>]
+    let ``one file - UnitTests`` () =
+        """
+#pragma warning disable 1591
+using System;
+using Microsoft.Quantum.Core;
+using Microsoft.Quantum.Intrinsic;
+using Microsoft.Quantum.Simulation.Core;
+using Microsoft.Quantum.Simulation.Simulators;
```

Comment:
From experience, it is better if instead of `using` statements, you actually use full-qualified-names for the types that you need from this namespaces.